src/recipe.yaml

```diff
@@ -370,11 +370,11 @@
     traffic:
       timeout: 300
     resources:
-      gpu: 2
+      gpu: 8
       gpu_type: nvidia-a100-80gb
   engine_config:
     model: nvidia/Llama-3.1-Nemotron-70B-Instruct-HF
-    max_model_len: 2048
```

Comment:
The reason for this is that we don't have 8 GPU to test it out.

but I think we can try 4. and set the context length to 64k for now

Reply:
@aarnphm The problem is that it blocks us from leveraging the whole context length when we run it on 8xA100. What would be your recommendation for running it?